lms/djangoapps/mobile_api/utils.py

```diff
@@ -58,3 +60,15 @@ def _decorator(func_or_class):
             func_or_class.permission_classes += (IsUser,)
         return func_or_class
     return _decorator
+
+
+def get_pagination(friends):
```

Comment:
Since this is specific to the facebook social API pagination design, this should go into a utility file within the social folder.